test/hotspot/gtest/nmt/test_vmatree.cpp

```diff
@@ -724,3 +724,19 @@ TEST_VM_F(NMTVMATreeTest, TestConsistencyWithSimpleTracker) {
     }
   }
 }
+
+TEST_VM_F(NMTVMATreeTest, CommitUseFlagInplace) {
```

Comment:
Change this test to use `expect_equivalent_form`, see other usages to see what should look like. You can add multiple checks even, so that we get how the range changes over time.

Reply:
Is the bug actually in the summary accounting? If so, I'd like to see a better test name here that indicates that and *not* change this to `expect_equivalent_form` and so on.

Reply:
New name chosen.